scripts/script-tag.js

```diff
@@ -142,7 +142,8 @@
             })
 
         } else if(location.pathname.includes('cart')) {
-            jQueryBopis("[data-cart-item-property-name]:contains('pickupstore')").closest('li').hide();
+            // jQueryBopis("[data-cart-item-property-name]:contains('pickupstore')").closest('li').hide();
+            jQueryBopis('.cart__text div:contains("pickupstore")').hide();
```

Comment:
Is it generic?

Reply:
No, its specific to this store.

Reply:
I will explore to find a generic way, so we don't need to update this code everytime the theme changes.

Reply:
Reverted the change.